samples/templates/default-azuredeploy.json

```diff
@@ -202,6 +202,7 @@
             "DicomFunctions__DurableTask__ConnectionName": "AzureWebJobsStorage",
             "DicomServer__Features__EnableOhifViewer": "[parameters('deployOhifViewer')]",
             "DicomServer__Features__EnableExtendedQueryTags": "true",
+            "DicomServer__Features__SupportNewBlobFormatForNewService": "true",
```

Comment:
I'm wondering if this also change environment variable for CI pipeline.

Reply:
Not the CI pipeline, CI pipeline app service is already created